plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -369,7 +371,10 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
         int debugPort = getDebugPort();
         setupDebugTarget(devServerRunConfiguration, launch, debugPort, monitor);
       }
-      IVMInstall vmInstall = getVMInstall(configuration);
+      
+      IJavaProject javaProject = JavaCore.create(modules[0].getProject());
```

Comment:
I thought about this. In that case, the local server is still going to use a single VM to run all of them.